tests/Faithlife.Utility.Tests/StringUtilityTests.cs

```diff
@@ -33,6 +33,8 @@ public void OrdinalStringComparisonExtensionMethods()
 			Assert.IsFalse("abc".StartsWithOrdinal("bc"));
 			Assert.IsFalse("abc".EndsWithOrdinal("ab"));
 			Assert.IsTrue("abc".EndsWithOrdinal("bc"));
+			Assert.IsTrue("abc".ContainsOrdinal("bc"));
+			Assert.IsFalse("abc".ContainsOrdinal("cd"));
```

Comment:
I tried adding a `Assert.IsTrue("ábá".Contains("a\u0301"));` line up in the "demonstrate the problem" section but was surprised to find that it was actually false in all four target frameworks. 🤷🏼‍♂️